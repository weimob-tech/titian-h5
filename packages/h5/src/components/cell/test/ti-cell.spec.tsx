import { newSpecPage } from '@stencil/core/testing';
import { TiCell } from '..';

describe('ti-cell', () => {
  it('renders', async () => {
    const page = await newSpecPage({
      components: [TiCell],
      html: `<ti-cell title="标题"></ti-cell>`,
    });
    expect(page.root).toMatchSnapshot();
  });
});
