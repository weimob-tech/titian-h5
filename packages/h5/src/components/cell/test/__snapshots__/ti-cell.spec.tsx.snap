// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ti-cell renders 1`] = `
<ti-cell clickable="" divider="" title="标题">
  <!---->
  <div class="titian-cell titian-cell-clickable titian-cell-divider">
    <div class="titian-cell-title-wrap">
      <div>
        <div class="titian-cell-title">
          标题
        </div>
      </div>
    </div>
    <div class="titian-cell-desc-wrap">
      <div class="titian-cell-desc-content">
      </div>
    </div>
  </div>
</ti-cell>
`;
